Makefile

```diff
@@ -23,3 +27,9 @@ clean:
 	rm -f test
 	rm -f *.o
 
+install: $(LIB)
+	install -d $(DESTDIR)/$(PREFIX)/lib/
```

Comment:
Thank you for the contribution!
But I think it should be `$(DESTDIR)$(PREFIX)/lib/` without a slash between `DESTDIR` and `PREFIX`. isn't it?

Reply:
Not sure, does `$(DESTDIR)` have a trailing `/`? I think an extra `/` won't make a difference, but either way seems to work fine for me!

Reply:
I mean, to match line 32-34, without the slash seems better

Reply:
Updated the PR!